### PR TITLE
MBM-48 feature | add magelight

### PR DESCRIPTION
**Pull request by solidwaterslayer**

### CODE REVIEW

**Comment by MattKirchh** [comment]
The Magelight needs a custom block class the implements the functionality from `onPlace` and `tick` from `BoomButton.java` in this PR (#17). You can leave it as scheduled for 60 ticks. That can be modified once it's implemented with it's associated spell.

**Review by MattKirchh — APPROVED** [review]

File: src/main/java/com/mineboundteam/minebound/registry/BlockRegistry.java [hunk 1]
```diff
@@ -29,6 +30,7 @@ public class BlockRegistry {
     public static final RegistryObject<Block> ENERGIZED_IRON_BLOCK = BLOCKS.register("energized_iron_block", () -> new Block(Properties.copy(Blocks.IRON_BLOCK)));
     public static final RegistryObject<Block> ILLUMINANT_STEEL_BLOCK = BLOCKS.register("illuminant_steel_block", () -> new Block(Properties.copy(Blocks.IRON_BLOCK).strength(7, 9).lightLevel(i -> 13)));
     public static final RegistryObject<Block> GILDED_DIAMOND_BLOCK = BLOCKS.register("gilded_diamond_block", () -> new Block(Properties.copy(Blocks.DIAMOND_BLOCK).strength(5, 9)));
+    public static final RegistryObject<Block> MAGELIGHT = BLOCKS.register("magelight", () -> new Magelight(Properties.copy(Blocks.GLOWSTONE)));
```
Comment (insertStringHere) [inline src/main/java/com/mineboundteam/minebound/registry/BlockRegistry.java:33]: I'm not sure if this should make the glowstone sound when breaking

Comment (insertStringHere) [reply src/main/java/com/mineboundteam/minebound/registry/BlockRegistry.java:33]: preferably anything not "hard", like wool or dry sponge. I don't know, though

**Review by insertStringHere — APPROVED** [review]

File: src/main/java/com/mineboundteam/minebound/block/registry/BlockRegistry.java [hunk 2]
```diff
@@ -31,9 +35,9 @@ public class BlockRegistry {
     public static final RegistryObject<Block> ENERGIZED_IRON_BLOCK = BLOCKS.register("energized_iron_block", () -> new Block(Properties.copy(Blocks.IRON_BLOCK)));
     public static final RegistryObject<Block> ILLUMINANT_STEEL_BLOCK = BLOCKS.register("illuminant_steel_block", () -> new Block(Properties.copy(Blocks.IRON_BLOCK).strength(7, 9).lightLevel(i -> 13)));
     public static final RegistryObject<Block> GILDED_DIAMOND_BLOCK = BLOCKS.register("gilded_diamond_block", () -> new Block(Properties.copy(Blocks.DIAMOND_BLOCK).strength(5, 9)));
+    public static final RegistryObject<Block> MAGELIGHT = BLOCKS.register("magelight", () -> new Magelight(BlockBehaviour.Properties.of(Material.WOOL, MaterialColor.NONE).lightLevel((blockState) -> 15)));
```
Comment (MattKirchh) [inline src/main/java/com/mineboundteam/minebound/block/registry/BlockRegistry.java:38]: Magelight should not be in between the alloy blocks.

**Review by MattKirchh — CHANGES_REQUESTED** [review]

**Review by MattKirchh — APPROVED** [review]

**Pull request merged**